### ISSUE TITLE
"Latest version" -links in files invalid

### ISSUE THREAD

**Issue by peterdd** [ADOdb/ADOdb#443]
```sh
grep -rin 'Latest version ' | grep -v sourceforge
```

```
adodb-errorhandler.inc.php:12: * Latest version is available at http://php.weblogs.com
adodb-errorpear.inc.php:12: * Latest version is available at http://php.weblogs.com
adodb-exceptions.inc.php:13: * Latest version is available at http://php.weblogs.com
drivers/adodb-informix.inc.php:12:* Latest version is available at http://php.weblogs.com
drivers/adodb-oci805.inc.php:12: * Latest version is available at http://php.weblogs.com
drivers/adodb-mssqlpo.inc.php:12:* Latest version is available at http://php.weblogs.com
rsfilter.inc.php:12: * Latest version is available at http://php.weblogs.com
session/old/adodb-session.php:11:  Latest version of ADODB is available at http://php.weblogs.com/adodb
session/old/adodb-session-clob.php:11:  Latest version of ADODB is available at http://php.weblogs.com/adodb
session/old/adodb-cryptsession.php:14:  Latest version of ADODB is available at http://php.weblogs.com/adodb
tests/testmssql.php:13: * Latest version is available at http://php.weblogs.com
tests/test4.php:13: * Latest version is available at http://php.weblogs.com
```

While the http://adodb.sourceforge.net -links at least are forwarded to adodb.org, the links to php.weblogs.com are not working anymore and weblogs.com tells to be a verisign owned domain now and the php subdomain does not exist anymore.

**Comment by dregad** [comment]
@mnewnham did you know that sed has an option to replace file contents inline ? e.g.

```
git grep -l php\.weblogs\.com | xargs sed -i "/php\.weblogs\.com/s//adodb\.org/g"
```

**Comment by dregad** [comment]
This (as well as a few other issues with links) is fixed in 5.20.13.

**Comment by mnewnham** [comment]
I didn't, or maybe I did and forgot, the last time I used sed was probably 1997......
